moodle/main.go

```diff
@@ -13,7 +15,7 @@ import (
 
 	clientset "github.com/cloud-ark/kubeplus-operators/moodle/pkg/client/clientset/versioned"
 	informers "github.com/cloud-ark/kubeplus-operators/moodle/pkg/client/informers/externalversions"
-	"github.com/cloud-ark/kubeplus-operators/moodle/pkg/signals"
+	"github.com/cloud-ark/kubeplus/postgres-crd/pkg/signals"
```

Comment:
Why postgres-crd? We should keep this in kubeplus-operator/moodle path

Reply:
Ah good catch, atom go-plus automatically does imports sometimes I didn't catch it.